README.md

```diff
@@ -7,6 +7,27 @@
 [Development guide](docs/GET_STARTED.md)
 
 ## Release notes
+[27/03/2020]  1.0.0 version:
+
+ - `@cennznet/api@1.0.0` (current stage) is working with [cennznet/cennznet:latest][cennznet/cennznet:latest];
+
+Changes made in this version:
+ 
+- Derived query for estimating fee is updated
+- New format to use fee exchange 
+    const feeExchange = {
+                        assetId: feeAssetId,
+                        maxPayment: '50000000000000000',
+                    };
+                    const transactionPayment = {
+                        tip: 2,
+                        feeExchange: {
+                            FeeExchangeV1: feeExchange,
+                        },
+                    };
+- CENNZX rpc call to get price queries
+- Removed plugnet dependencies
+- Support MultiSignature, registry, discontinue support to v1.
 
```

Comment:
As we discussed, we could ignore merging old release branches in, but could you also add release notes from alpha and beta versions, as which highlights what changes we added?

Reply:
Ya.. It contains notes from beta version.. notes from alpha already in the README file